typescript/utils/src/addresses.ts

```diff
@@ -372,3 +372,8 @@ export function ensure0x(hexstr: string) {
 export function strip0x(hexstr: string) {
   return hexstr.startsWith('0x') ? hexstr.slice(2) : hexstr;
 }
+
+export const fromHexString = (hexstr: string) =>
+  Buffer.from(strip0x(hexstr), 'hex');
+
+export const toHexString = (buf: Buffer) => ensure0x(buf.toString('hex'));
```

Comment:
These feel redundant with addressToBytes and bytesToProtocolAddress above. I realize they're are subtle differences. If we really want to keep these, please give them more descriptive names to clarify when these should be used instead

Reply:
moving to the strings utils (for arbitrary length strings)